analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/TestConstraints.cs

```diff
@@ -19,30 +19,34 @@
  */
 
 using SonarAnalyzer.SymbolicExecution;
+using SonarAnalyzer.SymbolicExecution.Constraints;
 
 namespace SonarAnalyzer.UnitTest.TestFramework.SymbolicExecution
 {
     internal class TestConstraint : SymbolicConstraint
     {
+        private readonly string name;
```

Comment:
This can go away once we have `ConstraintKindEx`

Reply:
I tried it but this doesn't work:
```cs
public static class ConstraintKindTest
{
    public const ConstraintKind First = (ConstraintKind)(-1);
    public const ConstraintKind Second = (ConstraintKind)(-2);
    public const ConstraintKind Dummy = (ConstraintKind)(-3);
    public const ConstraintKind Fake = (ConstraintKind)(-4);
}
```
This returns `-2` for `TestConstraint.Second`.